Sources/Sentry/Public/SentrySpanContext.h

```diff
@@ -16,21 +17,23 @@ NS_SWIFT_NAME(SpanContext)
 
 @property (nonatomic) BOOL sampled;
 
-@property (nonatomic, copy) NSString *_Nullable operation;
+@property (nonatomic, copy) NSString *operation;
 
 @property (nonatomic, copy) NSString *_Nullable spanDescription;
 
-@property (nonatomic, copy) NSString *_Nullable status;
+@property (nonatomic) SentrySpanStatus status;
 
 @property (nonatomic, readonly) NSMutableDictionary<NSString *, NSString *> *tags;
```

Comment:
This should be thread safe, but SentryEvent isnt. Should I start to make things thread safe or lets keep this way?

Reply:
it should be thread-safe

https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/SpanContext.java#L42

`SentryEvent` was never supposed to be modified concurrently, but that's not the case for Transactions

Reply:
Got it, will fix this in the next PR.